openbsd/i386 build fails

The problem is with golang 1.16.6 symbol [unix.RLIMIT_NPROC](https://pkg.go.dev/golang.org/x/sys/unix#pkg-constants) undefined on `openbsd/i386` (all fine on openbsd/amd64):
https://github.com/tklauser/go-sysconf/blob/v0.3.6/sysconf_openbsd.go#L22

From the build log (https://github.com/rclone/rclone/runs/3150179581?check_suite_focus=true#step:15:35), section "compile all architectures test":
```
2021/07/24 10:53:13 Failed to run [go build --ldflags -s -X github.com/rclone/rclone/fs.Version=v1.56.0-beta.5602.1fcdec087.merge -trimpath -o rclone-v1.56.0-beta.5602.1fcdec087.merge-openbsd-386/rclone -tags  ..]: exit status 2
2021/07/24 10:53:13 Command output was:
# github.com/tklauser/go-sysconf
Error: ../../../../go/pkg/mod/github.com/tklauser/go-sysconf@v0.3.6/sysconf_openbsd.go:22:28: undefined: unix.RLIMIT_NPROC
```

https://github.com/rclone/rclone/pull/5402 is pending on this issue.

--- a/go.mod
+++ b/go.mod
@@ -4,5 +4,5 @@ go 1.13
 
 require (
 	github.com/tklauser/numcpus v0.2.3
-	golang.org/x/sys v0.0.0-20210630005230-0f9fa26af87c
+	golang.org/x/sys v0.0.0-20210816074244-15123e1e1f71
 )

